WebDriverAgentLib/Routing/FBElement.h

```diff
@@ -52,6 +52,9 @@ NSString *wdAttributeNameForAttributeName(NSString *name);
 /*! Whether element is an accessibility container (contains children of any depth that are accessible) */
 @property (nonatomic, readonly, getter = isWDAccessibilityContainer) BOOL wdAccessibilityContainer;
 
+/*! Child elements */
+@property(copy) NSArray *children;
```

Comment:
This can crash code since `XCUIElement` does not implement it. I would remove it and make `getSnapshotAsXML` accept snapshots. In tests you force cast it from double to Snapshot